src/main/java/frc/robot/subsystems/shooter/ShooterBed.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems.shooter;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ShooterBed extends SubsystemBase
+{
+    private ShooterBedIO                       _io;
+    private final ShooterBedIOInputsAutoLogged _inputs        = new ShooterBedIOInputsAutoLogged();
+    private PIDController                      _bedFeedback;
+    private Rotation2d                         _angleSetpoint = null;
+    private Rotation2d                         _minBedAngle   = Constants.ShooterBed.MIN_BED_ANGLE;
+    private Rotation2d                         _maxBedAngle   = Constants.ShooterBed.MAX_BED_ANGLE;
+
+    public ShooterBed(ShooterBedIO io)
+    {
+        _io = io;
+
+        _bedFeedback = new PIDController(0.03, 0, 0); // FIXME: Set values, calibrate
+    }
+
+    @Override
+    public void periodic()
+    {
+        _io.updateInputs(_inputs);
+        Logger.processInputs("Shooter/Bed", _inputs);
+
+        if (_angleSetpoint != null)
+        {
+            _io.setVoltage(_bedFeedback.calculate(_inputs.bedAngle.getRadians(), _angleSetpoint.getRadians()));
+        }
+    }
+
+    public void setAngle(Rotation2d angleSetpoint)
+    {
+        _angleSetpoint = new Rotation2d(MathUtil.clamp(angleSetpoint.getRadians(), _minBedAngle.getRadians(), _maxBedAngle.getRadians()));
+    }
+
+    public void setAngleOffset(Rotation2d angleOffset)
+    {
+        _io.setAngleOffset(angleOffset);
+    }
```

Comment:
Let's make this a dashboard setting. Just like the min angle and the max angle. Rather than passing it off to the IO implementation class, we'll keep track of the offset here. We'll keep track of the offset using a `Rotation2d`, but the parameter to the function will likely be a double that represents the angle in degrees.

We'll also want to create a new method to get the angle with the offset applied. I'm betting it'll look something like this:

```java
public Rotation2d getAngle()
{
    return _inputs.bedAngle.minus(_angleOffset);
}
```